### DISCUSSION TITLE
How to switch to Chinese Input?

### DISCUSSION THREAD

**Discussion by skyucca** [barry-ran/QtScrcpy#924] [Q&A]
It doesn't work after I press Shift button to switch to Chinese Input, could anyone help me with that? Mobile Phone: Xiaomi, Android 14.

**Comment by rockyfang** [comment]
手机和电脑上都安装微信输入法，打开手机上的输入栏时在电脑输入即可同步显示在手机上。

**Comment by Warrior930213** [comment]
手机切换讯飞输入法小米版或者谷歌gboard输入法就行，但是gboard不符合使用习惯，需要适应下
